va/va.h

```diff
@@ -2547,6 +2549,23 @@ typedef struct _VAEncMiscParameterCustomRoundingControl
         uint32_t    value;
     }   rounding_offset_setting;
 } VAEncMiscParameterCustomRoundingControl;
+
+typedef struct _VAEncMiscParameterContextParameterAdjust
+{
+    union{
+        struct {
+            /** \brief indicate whether context priority changed */
+            uint32_t context_priority_update :1;
+            uint32_t reserved                :31;
+        } bits;
+        uint32_t value;
+    }flags;
+    /** \brief task priority adjustment dynamically */
```

Comment:
What does dynamically  mean here?

Reply:
It means the user can change priority per frame other than per stream.